src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -108,9 +126,10 @@ internal static class CsvAppFileIO {
             if(variableName.StartsWith("$", StringComparison.Ordinal)) {
                 variableName = variableName.Substring(1);
             }
+
             variableName = variableName.Replace('.', '_').Replace('@', '_').Replace('$', '_');
```

Comment:
I would suggest also adding `[]` here (or maybe dropping `$` and `@` - are they actually invalid chars? I think dollar at least isn't.).